tests/device_tests/test_msg_recoverydevice_slip39_advanced.py

```diff
@@ -63,33 +66,25 @@ def input_flow():
     assert client.features.pin_protection is False
     assert client.features.passphrase_protection is False
     assert client.features.backup_type is messages.BackupType.Slip39_Advanced
-    assert debug.state().mnemonic_secret.hex() == secret
+    if secret:
```

Comment:
instead of making this optional, please use the ADVANCED_20 secret in the `test_extra_share`

Reply:
Done in 6446cd2